src/test_workflow/test_cluster.py

```diff
@@ -11,8 +11,10 @@
 from typing import Any, Generator, List, Tuple
 
 from test_workflow.integ_test.service import Service
+from test_workflow.integ_test.service_opensearch import ServiceOpenSearch
```

Comment:
I see you also change OSD above but here you only set SeviceOpenSearch, do you also need to set up ServiceOpenSearchDashboards?

Thanks.

Reply:
mm.. I am not certain if there are the same expectation to ServiceOpenSearchDashboards tho, maybe I can check with @gaiksaya when she is back to work?

But if you are referring to the update I made on [src/test_workflow/integ_test/local_test_cluster_opensearch_dashboards.py](https://github.com/opensearch-project/opensearch-build/pull/3034/files#diff-be5ff0e69b8c84d6471f8ca99a9af4ad6093ca25142ae6ac1f9ba7ab5aa5fdce), the reson behind it is I touched the parent class, the TestCluster; hence, I need to upddate both OS and OSD accordinly to prevent error on validation. Thanks!!